source/fundamentals/typescript.txt

```diff
@@ -79,81 +84,119 @@ You can find a code snippet that shows how to specify a type for the ``FindCurso
 class in the
 :ref:`Find Multiple Documents Usage Example <node-driver-find-usage-example-code-snippet>`.
 
-Limitations
------------
+To view an example of a mutually recursive type, which is not supported by the
+preceding classes, see the :ref:`<node-driver-limitations-mutual-recursion>` section.
 
-.. _node-driver-typescript-limitations-dot-notation:
+.. _node-driver-limitations:
 
-The driver cannot infer the type of values with keys containing **dot
-notation**. Dot notation is a property access syntax for navigating BSON objects.
-Click on the tabs to see code snippets that highlight this behavior:
+Limitations For Driver Version {+version+}
```

Comment:
While reading through this section, I wasn't able to find the information about what the limitations are quickly. I wonder if having a bulleted list of limitations that linked to the sections would be useful. The bulleted list should summarize what the limitations are rather than contain category names (current subtitles).

Reply:
1. "For" is a preposition which shouldn't be capitalized in titles unless it's the first word
2. The preposition associated with "Limitations" is usually "of". Let me know if there's some specific exception in which "for" is more appropriate.